### PR TITLE
Fix name of link to cp-local-development

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ There are a few labels in particular that are very useful if you are new to Zen:
 * There is a [critical bugs milestone](https://github.com/CoderDojo/community-platform/milestones/Critical%20bugs) for the most important bugs.
 * Some bugs are smaller CSS issues that are marked low priority and so are not as important to work on. 
 
-### Priorities for the project
+### Priorities for the projectl
```
Comment (tangentfairy) [inline README.md]: Is the 1 here intentional?

Comment (raboof) [inline README.md]: :( my fault for not looking at the diff

Comment (tangentfairy) [inline README.md]: `git commit --amend --noedit` is your friend (After you `git add` this file again following the change you make)

**Comment by tangentfairy** [comment]
Looks good, thanks @raboof! :) :+1:

**Comment by raboof** [comment]
Thanks for merging! ;)

**Pull request merged**